CONTRIBUTING.md

```diff
@@ -1,29 +1,75 @@
 # Project Liberty Participation and Contribution Guidelines
 
-These technical guidelines apply to all [Project
+Welcome! We’re thrilled you’re interested in our collaborative work to build a 
+more equitable civic architecture for the next generation of the web.
+
+As a relatively new project, we’re moving quickly in a 
+quasi-[”rocket ship”](https://github.com/OpenTechStrategies/open-source-archetypes/blob/master/arch-rocket-ship-to-mars.ltx) 
+mode to get our open source technologies to market. We encourage your contributions 
+and questions! We're doing our best to set a good foundation for new contributors, 
+but please excuse us if we aren’t immediately responsive or if you find 
+anything lacking… (for things you find lacking, please let us know by submitting 
+an Issue or alerting us in the [Forums](https://forums.projectliberty.io/)).
+
+We will continue to update these contribution guidelines. For now, you will see 
+a few placeholders (and see Issues below for submitting a documentation related 
+request).
+
+These contribution guidelines apply to all [Project
 Liberty](https://ProjectLiberty.io/) development, although a given
 sub-project may extend or override some of these guidelines to suit
 its specific needs.
 
-Please see also the [Code of Conduct](CODE_OF_CONDUCT.md), which applies
-everywhere.
+A Few Useful Resources 
+- General Questions + Support: [See the Forums](https://forums.projectliberty.io/)
+- [Project Liberty Whitepaper on DSNP](https://unfinished.com/wp-content/uploads/dsnp_whitepaper.pdf)
+- [DSNP Roadmap](https://spec.dsnp.org/#implementation-status)
+- [DSNP Dev Portal](https://www.dsnp.org/dev-portal-introduction/)
+- Submit a bug: see Submitting Issues below
+
+## Code of Conduct
+
+Our [Code of Conduct](CODE_OF_CONDUCT.md) applies across all Project Liberty
+development.
+
+## How to Set Up a Development Environment 
+(coming soon....)
```

Comment:
The "(coming soon....)" can be replaced by permanent placeholder text, something like:

"The development environment setup guidelines for your particular repository within Project Liberty should go here.  For some examples, see ..."